script/install.sh

```diff
@@ -62,26 +62,43 @@ retry() {
         fi
     done
 }
-
 # url, path, [gpg signature url to verify]
+# url and the gpg signature can be an array of urls
```

Comment:
Let's change it to `path, verify, url...` then, and always require 0 or 1 for verify.

Reply:
The signature can be assumed to be path+`.sig`.

Reply:
Ok